### ISSUE TITLE
VoiceOver for mac OS: Announce state changes when a toggle button is activated

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#1140]
> Automatically generate confirmatory speech when toggling a "pressed" toggle button to the "not pressed" state. Currently, VoiceOver in macOS Safari is silent when carrying out this action, and the Community Group is concerned about the usability impact this could be having.

Thanks for the feedback. I will pass this on as well. One correction though:

> "VoiceOver in macOS Safari is silent when carrying out this action"

VO does speak "selected" when toggling the control on, and plays a press sound icon (or "earcon") when the state changes to either selected or unselected. 

_Originally posted by @cookiecrook in https://github.com/w3c/aria-at/issues/784#issuecomment-1257299177_

**Comment by mcking65** [comment]
@cookiecrook,  PR #1138 will change announcement of the not pressed state to optional, which will resolve #828.

However, there is still another related set of VoiceOver failures where we need to align. They are related to conveying state change when activating a toggle.

When activating a toggle, we assert that the screen reader must convey the state change. VoiceOver is only announcing a state change for one of the 6 commands.

Note that while VoiceOver does issue an earcon when toggling the state to not pressed with ctrl-opt-space, ctrl-opt-space creates that earcon all the time even it does nothing. So, that earcon does not convey any information other than the fact that the key was pressed.

Please see the report [VoiceOver for macOS and Safari for Toggle Button | ARIA-AT Reports](https://aria-at.w3.org/report/74559/targets/235#result-OTM5NeyIxMiI6NTM4fQmY2OD)

* Test 7: Operate a not pressed toggle button (7 passed, 2 failed)
* Test 8: Operate a pressed toggle button (6 passed, 3 failed)

Could you please run those two tests and confirm whether you get the same result that VO does not convey the state change?
